CMakeLists.txt

```diff
@@ -1,3 +1,26 @@
-cmake_minimum_required (VERSION 2.6)
-project (Hello)
-add_executable(Hello src/main.cpp)
+cmake_minimum_required(VERSION 2.6)
+project(basic_test)
+
+#GTEST
+ADD_SUBDIRECTORY (googletest-1.6.0)
+enable_testing()
+include_directories(${gtest_SOURCE_DIR}/include ${gtest_SOURCE_DIR})
+
+#MAIN
+include_directories(include)
+set(SOURCE_FILES src/Dollar.cpp src/main.cpp)
```

Comment:
You're setting the `SOURCE_FILES` variable but not using it, please erase this line so.